docs/language/accounts.mdx

```diff
@@ -156,6 +156,17 @@ to the `prepare` phase of the transaction.
   }
   ```
 
+  A script can get the `AuthAccount` for an account address
+  using the built-in `getAuthAccount` function:
+
+  ```cadence
+  fun getAuthAccount(_ address: Address): AuthAccount
+  ```
+
+  This function is only available in scripts; as they discard 
+  their changes upon completion. Attempting to use this function
+  outside of a script will cause a type error. 
+
```

Comment:
Can you add some clarification about what you can do with this `AuthAccount` object in the script? You can still perform all of the regular operations with it, right?